odoo-shippable/scripts/build-image.sh

```diff
@@ -422,7 +422,7 @@ PG_NON_DURABILITY=1 PG_LOGS_ENABLE=1 python ${REPO_REQUIREMENTS}/linit_hook/trav
 
 # Install & Configure RVM
 curl -sSL https://rvm.io/mpapis.asc | gpg --import -
-\curl -sSL https://get.rvm.io | /bin/bash -s stable --ruby
+\curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | /bin/bash -s stable --ruby
```

Comment:
This change is required for avoid this error`/bin/bash: line 880: __rvm_print_headline: command not found`

https://travis-ci.org/Vauxoo/docker-odoo-image/builds/245374522#L4590